### DISCUSSION TITLE
Publish @fluentui/react-northstar-prototypes

### DISCUSSION THREAD

**Discussion by axhl** [microsoft/fluentui#18329] [Ideas] [answered]
Hi, 

Per https://github.com/microsoft/fluentui/pull/16183#discussion_r539152180, `@fluentui/react-northstar-prototypes` is currently private. Making the package available would be a great help for prototyping and experimenting, despite breaking changes and other standard caveats. 

Would you be willing to publish it?

Thanks

**Comment by axhl** [comment]
Politely and with appreciation for your work tagging @layershifter to seek your thoughts on the above – would you be willing to publish `@fluentui/react-northstar-prototypes` ?

**Reply by layershifter** [reply]
Can you please clarify your use case? How it could improve your workflow?

**Reply by axhl** [reply]
Thanks for reply. We were seeking access to unreleased components (I _think_ the table specifically, but it's been awhile) for our own prototypes but ended up moving away from Fluent in favour of another library. We no longer have any need or use case for a published `prototypes`. Others might though.

**Reply by layershifter** [reply]
To be clear: via prototypes we don't provide new components, there we demonstrate their usage. That's I was a bit confused why it's needed 🙄

**Comment by layershifter** [comment, marked as answer]
For now we are not going to publish `@fluentui/react-northstar-prototypes`, but is there will be partner asks we can do it 😉
